### PR TITLE
Graalvm templates bugfixes

### PR DESCRIPTION
**Pull request by richarddd**
*Description of changes:*
Adds missing metadata and bugfix

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: provided.al2/graalvm/java17/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/build-image.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 set -e
 
-docker build -t al2-graalvm:gradle
\ No newline at end of file
+docker build -t al2-graalvm:gradle .
```
Comment (mndeveci) [inline provided.al2/graalvm/java17/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/build-image.sh:4]: Do we want to add this `build-image.sh` file into other GraalVM templates? I saw it is only in java17 ones.

Comment (hawflau) [reply provided.al2/graalvm/java17/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/build-image.sh:4]: I believe this change is needed in all four `build-image.sh` files.

Comment (hawflau) [reply provided.al2/graalvm/java17/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/build-image.sh:4]: note: the two under `maven` are correct. The one for java11 gradle needs to be fixed

Comment (mndeveci) [reply provided.al2/graalvm/java17/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/build-image.sh:4]: @hawflau So this is fixing the only problematic one then?

Comment (hawflau) [reply provided.al2/graalvm/java17/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/build-image.sh:4]: yep

File: provided.al2/graalvm/java11/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/samconfig.toml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 version=0.1
 [default.build.parameters]
-build_image = "al2-graalvm:gradle"
+build_image = ["al2-graalvm:gradle"]
```
Comment (hawflau) [inline provided.al2/graalvm/java11/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/samconfig.toml:3]: note: `--build-sam` is multiple as defined [here](https://github.com/aws/aws-sam-cli/blob/develop/samcli/commands/build/command.py#L108-L122). If `build_image` is provided as a string in `samconfig.toml`, `click` will parse it as a tuple and break down the string. e.g.

```
2022-06-30 14:54:50,806 | Expand command line arguments to:
2022-06-30 14:54:50,807 | --template_file=mypath/202206/java11-graalvm-gradle/template.yaml --use_container --build_image=('a', 'l', '2', '-', 'g', 'r', 'a', 'a', 'l', 'v', 'm', ':', 'g', 'r', 'a', 'd', 'l', 'e') --build_dir=.aws-
```

Comment (hawflau) [reply provided.al2/graalvm/java11/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/samconfig.toml:3]: We can consider fixing how SAM CLI reads and parse `samconfig.toml`. But it seems it's quickest to just fix `samconfig.toml` here

**Review by mndeveci — APPROVED** [review]

**Pull request merged**